lib/feedsme/index.js

```diff
@@ -199,9 +199,10 @@ Feedsme.prototype.triggerDep = async function triggerDep({ publish, promote, env
   // to be used for triggering dependent builds as well as the dependents for
   // the given package
   //
+
   const [releaseLine, dep] = await Promise.all([
-    publish ? this.release.get({ pkg: name }) : this.release.get({ pkg: name, version }),
-    Dependent.get(name)
+    publish ? this.release.get({ pkg: name }) : this.release.get({ pkg: name, version }), // TODO
```

Comment:
is somthing here still a TODO?

Reply:
left that note there thinking that this.release.get({ pkg: name }) might break because I thought version was required, but didn't see any tests breaking.

Reply:
@wjarvis1 it fetches the ReleaseHead when there's no version, its handled internally by the library so no need to TODO